### PR TITLE
Mimecast TI Regional

### PR DESCRIPTION
**Pull request by nipun-crestdatasystem**
This is a draft PR to run the validation checks for Mimecast TI Regional.

### CODE REVIEW

**Comment by v-prasadboke** [comment]
Hello @nipun-crestdatasystem, Thank you for marking this PR as Ready for review. 
This PR will be investigated and will update you about the same before 01 September, 2023.

Also please upload sample data to test the content of the solution.
Thanks

**Comment by v-prasadboke** [comment]
Hello @nipun-crestdatasystem, Can you please share working images of data connector and function app. 
Thanks.

**Comment by nipun-crestdatasystem** [comment]
> Hello @nipun-crestdatasystem, Can you please share working images of data connector and function app. Thanks.

@v-prasadboke can you please elaborate on working images, do you mean you need screenshots that indicate the successful execution of the data connector and function app?

**Comment by v-prasadboke** [comment]
> > Hello @nipun-crestdatasystem, Can you please share working images of data connector and function app. Thanks.
> 
> @v-prasadboke can you please elaborate on working images, do you mean you need screenshots that indicate the successful execution of the data connector and function app?

Yes @nipun-crestdatasystem, Results and and execution would be okay.
Thanks.

**Comment by nipun-crestdatasystem** [comment]
> > > Hello @nipun-crestdatasystem, Can you please share working images of data connector and function app. Thanks.
> > 
> > 
> > @v-prasadboke can you please elaborate on working images, do you mean you need screenshots that indicate the successful execution of the data connector and function app?
> 
> Yes @nipun-crestdatasystem, Results and and execution would be okay. Thanks.

**Comment by nipun-crestdatasystem** [comment]
> > > > Hello @nipun-crestdatasystem, Can you please share working images of data connector and function app. Thanks.
> > > 
> > > 
> > > @v-prasadboke can you please elaborate on working images, do you mean you need screenshots that indicate the successful execution of the data connector and function app?
> > 
> > 
> > Yes @nipun-crestdatasystem, Results and and execution would be okay. Thanks.

@v-prasadboke please find the attached images as requested by you.
image (11)
image (12)

**Comment by v-prasadboke** [comment]
Hello @nipun-crestdatasystem, Thank you for committing the requested changes. You can expect an update till 11 September, 2023.

File: Tools/Create-Azure-Sentinel-Solution/V2/WorkbookMetadata/WorkbooksMetadata.json [hunk 1]
```diff
@@ -5511,5 +5511,20 @@
     "templateRelativePath": "MailGuard365Dashboard.json",
     "subtitle": "",
     "provider": "MailGuard 365"
+},
+{
+    "workbookKey": "MimecastTIRegionalWorkbook",
+    "logoFileName": "Mimecast.svg",
+    "description": "A workbook providing insights into Mimecast Regional Threat indicator.",
+    "dataTypesDependencies": ["ThreatIntelligenceIndicator"],
```
Comment (v-prasadboke) [inline Tools/Create-Azure-Sentinel-Solution/V2/WorkbookMetadata/WorkbooksMetadata.json:5519]: Hello @nipun-crestdatasystem, Please add dataconnector dependency property in the workbook metadata. You refer the workbook metadata declared above.

File: Solutions/MimecastTIRegional/SolutionMetadata.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+    "publisherId": "mimecast",
+    "offerId": "azure-sentinel-solution-mimecasttiregional",
+    "firstPublishDate": "2022-02-24",
```
Comment (v-prasadboke) [inline Solutions/MimecastTIRegional/SolutionMetadata.json:4]: Is the first published data ok??

Comment (nipun-crestdatasystem) [reply Solutions/MimecastTIRegional/SolutionMetadata.json:4]: I will update that actually the was raised previously on the mentioned date but it was closed without merging.

**Comment by v-prasadboke** [comment]
Hello @nipun-crestdatasystem, Thanks for committing the requested changes. You can expect an update till 18 September, 2023.

**Review by v-atulyadav — APPROVED** [review]

**Comment by nipun-crestdatasystem** [comment]
@v-atulyadav the link for deploying the data connector is not correctly pointing to the target in this file - azuredeploy_MimecastTIRegional_AzureFunctionApp.json can you please update it to the right location?

**Pull request merged**